CHANGELOG.md

```diff
@@ -2,11 +2,7 @@
 
 #### Features
 
-* Your contribution here.
-
-#### Fixes
-
-* Your contribution here.
+* [#431](https://github.com/ruby-grape/grape-swagger/pull/431)Add summary and deprecated to the operation object generator to be more compliant with [OpenAPI v2](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#operation-object) - [@aschuster3](https://github.com/aschuster3)
```

Comment:
Please make this the same syntax as everything else. Missing `:` after the issue number and a period at the end.